requirements-dev.txt

```diff
@@ -0,0 +1,79 @@
+#
+# This file is autogenerated by pip-compile with python 3.7
```

Comment:
this says 3.7 but the pre-commit config is 3.8?

Reply:
Thank you!